northstar_tests/test_container/src/main.rs

```diff
@@ -39,30 +40,36 @@ enum TestCommands {
         path: PathBuf,
     },
     Whoami,
-    Loop,
+    LeakMemory,
```

Comment:
what happened to loop?

Reply:
`Loop` was unused. It's all `Io` and `Os` (nix) erros.